datasets/diplomacy_detection/README.md

```diff
@@ -0,0 +1,100 @@
+---
+YAML tags:
+annotations_creators: [no-annotation]
```

Comment:
Actually there are annotations: the truthful / deceptive labels, which are generated by players (see https://github.com/DenisPeskov/2020_acl_diplomacy/tree/master/data). I would suggest `crowdsource` or `found` here. @lhoestq WDYT ?

Reply:
I would say `found` since they're part of the data collection (same process)
```suggestion
annotations_creators:
- found
```